pages/index.js

```diff
@@ -56,11 +61,11 @@ const CategoryBox = styled.span`
 `;
 
 const ListItem = styled.li`
-  position: relative;
+  display: flex;
+  justify-content: space-between; /* Space between the item content and the category box */
+  align-items: center; /* Center content vertically */
```

Comment:
Good improvement!

Reply:
Thank you. I've also taken out the comments now. :)